src/Corax/Mappings/IndexFieldsMappingBuilder.cs

```diff
@@ -76,14 +72,14 @@ public IndexFieldsMappingBuilder AddDynamicBinding(Slice fieldName, FieldIndexin
     }
     
     public IndexFieldsMappingBuilder AddBinding(int fieldId, string fieldName, Analyzer analyzer = null, bool hasSuggestion = false,
-        FieldIndexingMode fieldIndexingMode = FieldIndexingMode.Normal, bool hasSpatial = false)
+        FieldIndexingMode fieldIndexingMode = FieldIndexingMode.Normal, bool hasSpatial = false, bool shouldStore = false)
```

Comment:
can we keep the same order of properties? shouldStore should be before hasSpatial

Reply:
What is the logic? What is the previous order of properties?

Reply:
In all the method in this class you have:
```
        FieldIndexingMode fieldIndexingMode = FieldIndexingMode.Normal, bool shouldStore = false, bool hasSpatial = false)
```

and in this one it s the other way around